bitrise.yml

```diff
@@ -0,0 +1,93 @@
+---
+format_version: '11'
```

Comment:
@R15B Since this is an open source project, could we keep the yml file in Bitrise directly? For example for the UI SDK we've been using Bitrise for a while and this was an option. Adding the yml file to the repo wasn't mandatory.